app/views/additional_details/submissions/_submission_info.html.haml

```diff
@@ -9,6 +9,10 @@
             %p.list-group-item-text
               - if item.item_type == 'multiple_dropdown'
                 = qr.content.delete('[]').delete('\"')
+              - elsif item.item_type == 'time'
+                = convert_time(qr.content)
+              - elsif item.item_type == 'checkbox'
+                = qr.content.gsub(/["\[\]]/, '')
```

Comment:
I tested `qr.content.delete('[]').delete('\"')` and `qr.content.gsub(/["\[\]]/, '')` and they appear to be doing the same thing: Replacing/removing the characters []". We should combine this block and the if block into a single statement. 
`if ['checkbox', 'multiple_dropdown'].include?(qr.content)`
`  qr.content.delete('[]\"')` or `qr.content.gsub(/["\[\]]/, '')`